src/main/java/org/cbio/gdcpipeline/GDCPipelineApplication.java

```diff
@@ -15,24 +15,23 @@
 import java.util.ArrayList;
 import java.util.List;
 
-
+/**
+ * @author Dixit Patel
+ */
 @SpringBootApplication
 public class GDCPipelineApplication {
-
-
 	private static Log LOG = LogFactory.getLog(GDCPipelineApplication.class);
-
 	private final static String GDC_JOB = "gdcJob";
 	private final static String DEFAULT_DATATYPES = "ALL";
 	private final static String DEFAULT_FILTER_SAMPLE = "true";
 
-
-	private static Options getOptions(String []input){
+    private static Options getOptions(String []input){
 		Options options = new Options();
         options.addRequiredOption("s","source",true,"source directory for files");
 		options.addRequiredOption("o","output",true,"output directory for files");
-		options.addRequiredOption("study", "study", true, "Cancer Study Id");
-		options.addOption("filter_sample", "filter_sample", true, "True or False. Flag to filter Normal samples. Default is True ");
+		options.addRequiredOption("cancer_study_id", "cancer_study_id", true, "Cancer Study Id");
+		options.addRequiredOption("manifest_file","manifest_file",true,"Manifest file path");
+        options.addOption("filter_sample", "filter_sample", true, "True or False. Flag to filter Normal samples. Default is True ");
```

Comment:
done. I thought single characters would be more confusing as the list of options grow